### DISCUSSION TITLE
没事了

### DISCUSSION THREAD

**Discussion by xkitme** [Meloong-Git/PCL#4588] [提问] [answered]

**Comment by wuliaodexiaoluo** [comment, marked as answer]
版本文件的删除按钮是移动到回收站
如果你点的是版本列表右侧，针对删除整个 .minecraft 的选项，这类删除操作已经有警告提示了，并且已经采取措施保证用户并非无意识/下意识选择
针对文件资源管理器的等第三方软件删除，这类拦截需要调用驱动获取更高级别的内核态管理权限，并且实现起来相当复杂，每一步都需要反复校验操作，一旦失误就会导致蓝屏，故不可能考虑

**Comment by Pigeon0v0** [comment]
不要把 Discussion 的内容删除，这会造成一定的困扰。

解决了直接关闭即可。
